package.json

```diff
@@ -62,7 +62,6 @@
     "site:build:install": "cd site && npm install --no-audit",
     "site:build:assets": "cd site && npm run build",
     "postinstall-pack": "node ./scripts/postinstall.js",
```

Comment:
This `postinstall-pack` hook gets renamed to `postinstall` in a script triggered from the `prepublishOnly` hook, so that when _users_ install this package they see some output we want them to see at install time.